src/obr/core/core.py

```diff
@@ -142,6 +142,32 @@ def logged_func(func, doc, **kwargs):
     doc["history"].append(res)
 
 
+def get_mesh_stats(owner_path: str) -> dict:
+    """Check constant/polyMesh/owner file for mesh properties
+    and return it via a dictionary"""
+    nCells = None
+    nFaces = None
+    if Path(owner_path).exists():
+        with open(owner_path, "r", errors="replace") as fh:
```

Comment:
Like in the other PR, "r" is the default.
```suggestion
        with open(owner_path, errors="replace") as fh:
```

Reply:
Good catch! Back in the days of python 2 it wasn't.